### PR TITLE
Movecontent

### PR DESCRIPTION
**Pull request by iregina**
Moving content from old docs to new one

### CODE REVIEW

**Comment by iregina** [comment]
I'm not sure why all the vercel checkes are failing @MartinMinkov - do you know why?

File: sidebars.js [hunk 1]
```diff
@@ -94,11 +94,15 @@ module.exports = {
...
       ],
     },
-
     {
       type: "category",
-      label: "Exchange Operators",
-      items: ["exchange-operators/exchange-faq"],
+      label: "Node Developers",
+      items: [
+        "node-developers/overview",
```
Comment (jasongitmail) [inline sidebars.js:101]: @iregina appears to be a typo, missing a closing `}` for Node Developers

Comment (iregina) [reply sidebars.js:101]: Fixed! Thanks.

**Review by jasongitmail — APPROVED** [review]
This is not complete, but we've discussed and are going to merge this big PR, so that others can work in parallel with lower chance of merge conflicts. Pending any final push from @iregina of local items you want to include here

**Pull request merged**